Mapsui.Geometries/ArrowLineString.cs

```diff
@@ -82,7 +76,7 @@ public ArrowLineString(double[] startPoint, double[] endPoint, double degreeAngl
         ///     The length of this ArrowLine, as measured in the spatial reference system of this ArrowLine.
         /// </summary>
         private double? length;
-        public override double Length => length.HasValue ? length.Value : (length = StartPoint.Distance(EndPoint)).Value;
+        public double Length => length.HasValue ? length.Value : (length = StartPoint.Distance(EndPoint)).Value;
```

Comment:
Since we removed the override, we should also remove the virtual keyword from MultiLineString.cs.

Reply:
fixed